docs/website/docs/building-from-source/riscv.md

```diff
@@ -157,11 +157,10 @@ The SIMD code can be generated following the
 [IREE CPU flow](../guides/deployment-configurations/cpu.md)
 with the additional command-line flags
 
-```shell hl_lines="3 4 5 6 7 8"
+```shell hl_lines="3 4 5 6"
```

Comment:
FYI: you can use line ranges like `3-6`: https://squidfunk.github.io/mkdocs-material/reference/code-blocks/#highlighting-specific-lines-line-ranges